src/components/login/ProviderLink.tsx

```diff
@@ -74,11 +50,11 @@ const ProviderLink: FunctionComponent<Props> = ({
     }
   }, [clicked, provider.method, t])
 
-  const loginText = t(`login-with-provider`, { provider: loginMethod(provider) })
+  const loginText = t(`login-with-provider`, { provider: `${provider.name}` })
```

Comment:
```suggestion
  const loginText = t(`login-with-provider`, { provider: provider.name })
```

Don't need the enclosing template literal here